src/components/OrderTnrisDataForm.jsx

```diff
@@ -636,7 +646,7 @@ class OrderTnrisDataForm extends Component {
             This dataset has been added to the shopping cart.
           </p>
           <p className="mdc-typography--body2">
-            Please visit the shopping cart to finalize your order.
+            Please visit the <button style={{color: '#1E8DC1'}} className="mdc-button" onClick={this.handleOrderCartView}>shopping cart</button> to finalize your order.
```

Comment:
I am seeing this warning in the console when I click the text to navigate to the cart: submission canceled because the form is not connected

I think the browser is defaulting the button to type="submit" since its type hasn't been declared implicitly and it is in a form.
https://www.w3schools.com/tags/att_button_type.asp

add the attributetype="button" to the button element to prevent this warning on click